ui/src/main/webapp/lib/graphql-errors/index.js

```diff
@@ -10,11 +10,9 @@ const friendlyMessage = {
   FAILED_PERSIST: 'An error occurred while trying to save the configuration',
```

Comment:
I wouldn't mind getting rid of `FAILED_PERSIST` and replacing it with `FAILED_CREATE`, `FAILED_UPDATE`, `FAILED_DELETE` since that's the direction our mutations are going.